pkg/agent/run.go

```diff
@@ -105,21 +114,32 @@ func Run(cmd *cobra.Command, args []string) {
 		// what "ready" means for the agent, we just return 200 OK inconditionally.
 		// The goal is to satisfy some Kubernetes distributions, like OpenShift,
 		// that require a liveness and health probe to be present for each pod.
+		log.Info("Healthz endpoints enabled", "path", "/healthz")
 		server.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(http.StatusOK)
 		})
+		log.Info("Readyz endpoints enabled", "path", "/readyz")
```

Comment:
Nit, unimportant: weird that we start a readiness and liveness server when running `--one-shot`

Reply:
I agree it's weird, perhaps in a future PR we can make it so that the API server is not started in one-shot mode.